test/unit/alphabet/alphabet_test.cpp

```diff
@@ -88,6 +88,8 @@ TYPED_TEST(alphabet, default_value_constructor)
 
 TYPED_TEST(alphabet, assign_rank)
 {
+    if (std::is_same_v<TypeParam, phred42> || std::is_same_v<TypeParam, phred63> || std::is_same_v<TypeParam, phred68>)
+        return;
```

Comment:
why are these not being tested?

Reply:
Will violate the assertions when tested with full char range, I restricted now the loop range for the phred types.

Reply:
But the loop only checks valid ranks! This should behave just like the other alphabets, or not?